<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModbusKsuManualSelectiveTestingWidget</class>
 <widget class="QWidget" name="ModbusKsuManualSelectiveTestingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="requestGroupBox">
       <property name="title">
        <string>Запрос</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="requestReadButton">
          <property name="text">
           <string>Прочитать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="requestWriteButton">
          <property name="text">
           <string>Записать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="requestClearButton">
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="settingsGroupBox">
       <property name="title">
        <string>Настройка</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="loadSettingsButton">
          <property name="text">
           <string>Загрузить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveSettingsButton">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearSettingsButton">
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
